deployment/helm/topology-aware/README.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+This chart deploys topology-aware Node Resource Interface (NRI) plugin. Topology-aware NRI
+resource policy plugin is a NRI plugin that will apply hardware-aware resource allocation
+policies to the containers running in the system.
+
+# Installing the Chart
+
+Path to the chart: `nri-resource-policy-topology-aware`.
+
+```
+helm repo add nri-plugins https://containers.github.io/nri-plugins
+helm install my-topology-aware nri-plugins/nri-resource-policy-topology-aware --namespace kube-system
+```
+
+The command above deploys topology-aware NRI plugin on the Kubernetes cluster within the
+`kube-system` namespace with default configuration. 
+
+# Uninstalling the Chart
+
+To uninstall the topology-aware plugin run the following command:
+
+```
+helm delete my-topology-aware --namespace kube-system
+```
+
+# Configuration options
+
+The tables below present an overview of the parameters available for users to customize with their own values,
+along with the default values.
+
+| Name                     | Default                                                                                                                       | Description                                          |
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`             | [ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware](ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware)    | container image name                     |
```

Comment:
The link should be an URL, like `[ghcr.io/containers/...](https://ghcr.io/containers/...)`